panel/auth.py

```diff
@@ -354,15 +377,22 @@ def _on_error(self, response, body=None):
             log.warning(f"{provider} OAuth provider failed to fully "
                         f"authenticate returning the following response:"
                         f"{body}.")
-        raise HTTPError(500, f"{provider} authentication failed {body}")
+        self.set_header("Content-Type", 'text/html')
+        self.write(self._error_template.render(
+            npm_cdn=config.npm_cdn,
+            title='Panel: Authentication Error',
+            error_type='Authentication Error',
+            error=body.get('error', 'Unknown Error'),
+            error_msg=body.get('error_description', body)
+        ))
 
 
 class GenericLoginHandler(OAuthLoginHandler, OAuth2Mixin):
```

Comment:
Isn't there a way to update the scope other than defining `PANEL_OAUTH_SCOPE`? we would like to have it as a config option the same the URLs are

Reply:
Indeed, should expose it this way.